images/virt-launcher/werf.inc.yaml

```diff
@@ -114,6 +117,7 @@ shell:
     seabios \
     libffi8 \
     swtpm-tools \
+    glibc \
```

Comment:
Seems redundant. glibc should be installed in the base image.

Reply:
You're right.
Fixed.